_posts/2021-11-10-release-0.9.13.md

```diff
@@ -0,0 +1,194 @@
+---
+layout: post
+comments: true
+title:  "CARLA 0.9.13 Release"
+subtitle: "User defined traffic manager paths, carla-agents Python package, user defined pedestrian animations and new vehicle behaviours"
+description: "User defined traffic manager paths, carla-agents Python package, user defined pedestrian animations and new vehicle behaviours"
+author: "@MattRowe18"
+image: 'img/carla.jpg'
+background: '/img/posts/2021-08-02/town10.png'
+---
+
+{% include youtube.html id="tWAg1fIDNCg" %}
+
+{% include release_button.html release_tag="0.9.13" %}
+
+The __0.9.13 release of CARLA__ is finally here, so read on to see how the new improvements will revolutionize your workflow. 
+
+The most exciting new feature is the [__instance semantic segmentation__](#instance-semantic-segmentation)! A new camera sensor can be used to output images with the semantic tag and a unique ID for each world object embedded in the channels of the image file. Now the ground truth ID is available to differentiate objects of the same type in the camera field of view for training and evaluating networks to differentiate  and track overlapping objects of the same class. 
+
+[__Runtime texture streaming__](#runtime-texture-streaming) is now possible through the CARLA API. This new functionality enables the rendering material of objects in the scene to be updated at runtime without requiring the Unreal Editor. This can help produce training data to avoid overfitting pitfalls.
+
+CARLA's Python API now provides new functions to retrieve [__pedestrian bone positions__](#pedestrian-ground-truth-skeletons) and hence evaluate the pose, facilitating the evaluation and training of pose estimation models for use in autonomous driving and also adding custom gestures to standard animations through the API. 
+
+CARLA 0.9.13 presents new assets to add to your simulations. [__4 new pedestrians__](#new-pedestrians) are provided including 2 children! The CARLA garage now has a [__brand new vehicle__](#new-volkswagen-van), the [__Volkswagen T2 2021__](#new-volkswagen-van). All generation 2 cars now include [__opening doors__](#opening-doors) that can interact with the environment through collisions. Especially useful for training and testing autonomous vehicles in car parks or on tight city streets. 
+
+This release comes with major improvements to the traffic manager, with new logic to promote [__more realistic vehicle behavior at intersections and at high speeds__](#improved-traffic-manager-logic). The API now offers new functionality to guide vehicles with a __user defined route or path__ and __vehicle lights__ now react to weather conditions and junctions. 
+
+__installation improvements:__ following a continued commitment to improve the useability of the CARLA simulator, the CARLA agents wheel is now __available through PyPi__ so can now be installed with pip. A new wheel for the core CARLA API library is now also available for Python version 3.9.
```

Comment:
> the CARLA agents wheel is now __available through PyPi__ so can now be installed with pip

At the end, the agents API won't be available at Pypi, so this should only includes the new Python 3.9 wheels

Reply:
removed